client/src/common/components/SelectBox.tsx

```diff
@@ -9,24 +9,27 @@ import {
   Option,
 } from '../style/style';
 const SelectBox = ({
+  selectedValue,
+  setSelectedValue,
   selectOptionData,
   selectDefaultOption,
 }: {
-  selectOptionData: SelectOption[];
+  selectedValue?: string;
+  setSelectedValue?: React.Dispatch<React.SetStateAction<string>>;
+  selectOptionData: SelectOption[] | undefined;
```

Comment:
selectOptionData? SelectOption[] 으로 적어도 될 듯 합니다
```suggestion
  selectOptionData?: SelectOption[];
```

Reply:
네! 수정하겠습니다~!